typedb-src/modules/ROOT/pages/quickstart-guide.adoc

```diff
@@ -246,11 +243,12 @@ match
 get $e;
 ----
 
+Retrieves all emails owned by every person with a full name that contains "Kevin".
```

Comment:
```suggestion
Retrieve all emails owned by each person with a full name that contains "Kevin" (we have only one such person in the example database):
```

Reply:
I generally try to use less parenthesis. This particular statement seems good enough to be a standalone sentence, IMHO. Do you think it is better to make it more vague about the IAM and not a standalone sentence?

Reply:
I experimented with different ways to do this and settled with your variant.